cmd/spire-agent/cli/cli.go

```diff
@@ -1,16 +1,17 @@
 package cli
 
 import (
-	"log"
+	stdlog "log"
 
 	"github.com/mitchellh/cli"
 	"github.com/spiffe/spire/cmd/spire-agent/cli/api"
 	"github.com/spiffe/spire/cmd/spire-agent/cli/healthcheck"
 	"github.com/spiffe/spire/cmd/spire-agent/cli/run"
+	"github.com/spiffe/spire/pkg/common/log"
 	"github.com/spiffe/spire/pkg/common/version"
 )
 
-func Run(args []string) int {
+func Run(args []string, logOptions ...log.Option) int {
```

Comment:
It feels a bit strange to pass cli args and also log options to Run. Perhaps it is worth considering a `CLI` struct on which the options can be set, similar to how you've done with the Command struct? Our main.go could then become something like `os.Exit(new(cli.CLI).Run(os.Args[1:]))`

Reply:
I think that makes sense.

Reply:
done